sdk/src/test/scala/com/horizen/account/receipt/EthereumConsensusDataReceiptTest.scala

```diff
@@ -0,0 +1,32 @@
+package com.horizen.account.receipt
+
+import com.horizen.account.receipt.EthereumConsensusDataReceipt.addDataToBloomFilter
+import com.horizen.utils.BytesUtils
+import org.junit.Assert._
+import org.junit._
+import org.scalatestplus.junit.JUnitSuite
+import org.scalatestplus.mockito._
+
+class EthereumConsensusDataReceiptTest
+    extends JUnitSuite
+    with MockitoSugar
+    with ReceiptFixture {
+
+  @Test def bloomFilterTest(): Unit = {
```

Comment:
We need more tests, more regression data.
I believe that Geth have enough tests for bloom filters we need to reuse all of them.

Reply:
There are not too many tests in geth regarding bloom filter, but I guess they are enough as a slight variance would immediately fail the test. I mean, their main test just adds 100 elements to the bloom filter, gets the resulting data and compares its hash to a precalculated value. This would fail if a single bit of the bloom filter is off.

We should just do the same, preferably exactly the same, because then we can also reuse their precalculated hash value and make sure it matches.

@GitMark0 see `TestBloom` and `TestBloomExtensively` in geth source: `core/types/bloom9_test.go`.

Reply:
Geth tests are included now, I think this is ok :+1: